src/components/shared/MarkerTooltipContents.js

```diff
@@ -64,6 +67,14 @@ export default class MarkerTooltipContents extends PureComponent {
             {marker.title || marker.name}
           </div>
         </div>
+        {threadName
+          ? <div className="tooltipDetails">
+              <div className="tooltipLabel" key="thread">
+                Thread:
+              </div>
+              {threadName}
+            </div>
+          : null}
```

Comment:
I think that makes sense, I see the distinction between a marker property and the actual payload.